plugins/gateway/kong/deploy/kongPlugin.yaml

```diff
@@ -0,0 +1,7 @@
+apiVersion: configuration.konghq.com/v1
+kind: KongPlugin
+metadata:
+  name: apiclarity-plugin
+plugin: kong-plugin
+config:
+  host: "apiclarity.apiclarity:9000"
```

Comment:
please allow it to be configured with an env var during the deploy script (like `KONG_PROXY_CONTAINER_NAME`) - it will help for the helm support